must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -114,3 +113,22 @@ echo "collecting dump of oc get obc all namespaces" | tee -a  "${BASE_COLLECTION
 echo "collecting dump of oc get volumereplication all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get volumereplication --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/vr_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect volumereplication --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# For StorageSystem of all namespaces
+echo "collecting dump of oc get storagesystem all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+{ oc get storagesystem --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/ss_all_namespaces"
+{ oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect storagesystem --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# For oc logs odf-operator pods
+echo "collecting oc command odf-operator" | tee -a "${BASE_COLLECTION_PATH}/gather-debug.log"
+operator_pod=$(oc get pods -n openshift-storage| grep odf-operator | awk '{print $1}')
+LOG_DIR=${BASE_COLLECTION_PATH}/namespaces/logs/${INSTALL_NAMESPACE}
+mkdir -p "${LOG_DIR}"
+{ timeout 120 oc -n "${INSTALL_NAMESPACE}" logs --all-containers "${operator_pod}" &> "${LOG_DIR}"/"${operator_pod}".log; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
+
+# For oc logs odf-console pods
+echo "collecting oc command odf-console" | tee -a "${BASE_COLLECTION_PATH}/gather-debug.log"
+console_pod=$(oc get pods -n openshift-storage | grep odf-console | awk '{print $1}')
+LOG_DIR=${BASE_COLLECTION_PATH}/namespaces/logs/${INSTALL_NAMESPACE}
+mkdir -p "${LOG_DIR}"
+{ timeout 120 oc -n "${INSTALL_NAMESPACE}" logs --all-containers "${console_pod}" &> "${LOG_DIR}"/"${console_pod}".log; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
```

Comment:
Is there no other place where we collect Pod logs? All must-gather output I've seen has included the logs of all Pods in the namespace, why is this required here?

Reply:
No, for namespace we don't collect pod logs elsewhere. We have it in ceph and nooba for their resources.

Reply:
This statement seems absolutely wrong. Unless I'm missing something, this looks to me like we collect the logs of all Pods in the openshift-storage Namespace for every must-gather run: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_ocs-operator/1224/pull-ci-openshift-ocs-operator-master-ocs-operator-bundle-e2e-aws/1431303805407858688/artifacts/ocs-operator-bundle-e2e-aws/ocs-must-gather/artifacts/ocs-must-gather/registry-build02-ci-openshift-org-ci-op-s10lq9fp-stable-sha256-2cabcf61a08025e75eba8b82626357ddb48a44bf609f7d1483242ad5d0b1f5f0/namespaces/openshift-storage/pods/

I don't see anywhere in our scripts where we explicitly do this, so maybe it's part of the `oc adm inspect` of the namespace? In any case there's no reason to isolate these two specific Pods when we just want all logs in the namespace anyway.